activiti-spring-boot-starter/src/test/resources/processes/json-vars-extensions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id":"jsonVarsProcess",
+  "name":"myProcess.bpmn",
+  "extensions":{
+    "properties":{
+      "d440ff7b-0ac8-4a97-b163-51a6ec49faa1":{
+        "id":"d440ff7b-0ac8-4a97-b163-51a6ec49faa1",
+        "name":"var1",
+        "type":"json",
```

Comment:
@ryandawsonuk we agreed on no "type":"json" right?

Reply:
Ah yeah I'd forgotten to include that - just pushed https://github.com/Activiti/Activiti/pull/2212/commits/237e5d59a8b91e9b72aeac771da0d9c8701b2082